src/phoenix/session/client.py

```diff
@@ -29,20 +28,15 @@ def __init__(
         self,
         *,
         endpoint: Optional[str] = None,
-        use_active_session_if_available: bool = True,
+        **kwargs: Any,  # for backward-compatibility
```

Comment:
We should add a deprecation warning if someone passes in `use_active_session_if_available`.

Reply:
ok